packages/sanity/src/_internal/cli/commands/backup/backupGroup.ts

```diff
@@ -0,0 +1,26 @@
+import {CliCommandGroupDefinition} from '@sanity/cli'
+
+// defaultApiVersion is the backend API version used for dataset backup.
+export const defaultApiVersion = 'vX'
```

Comment:
Should this not be a proper API version instead of `vX`? If it's intentional to keep it at vX, can you add a comment explaining why and what it would take to move it to an official version?

Reply:
This should be released with `vX` Content Lake API version. Added the reason inline:
```
// First version of the backup API is vX since this feature is not yet released
// and formal API documentation is pending.
```